jobs/integration/validation.py

```diff
@@ -59,7 +59,7 @@
     if cpu_arch not in ['s390x', 'arm64', 'aarch64']:
         await validate_keystone(model)
     assert_no_unit_errors(model)
-    if ('vault' in model.applications and
+    if ('validate-vault' == model.info.name and
```

Comment:
Should be `in` not `==`, given that model name is generally `validate-vault-<uuid>`:
https://github.com/charmed-kubernetes/jenkins/blob/ceba5490a35ac806b237ed7fc247e846ad0cc0f7/jobs/validate-vault.yaml#L14-L15
https://github.com/charmed-kubernetes/jenkins/blob/ceba5490a35ac806b237ed7fc247e846ad0cc0f7/jobs/validate-vault/Jenkinsfile#L3